components/Table/Table.tsx

```diff
@@ -1,4 +1,5 @@
 import { styled, VariantProps } from '../../stitches.config';
+import { elevationVariants } from '../Elevation';
 import { Label } from '../Label';
 
 export const Caption = styled('caption', {
```

Comment:
@paulocfjunior We didn't update table `Caption` styles. To me the `Caption` should the table's title.

Mockups aren't integrating it at all.

I propose that we take our existing usage and apply its styles to the `Caption`. We could inherit from `Heading` for example.

WDYT?

Reply:
Yes, sounds good to me 👌